packages/teams-js/test/public/media.spec.ts

```diff
@@ -838,11 +822,58 @@ describe('media', () => {
           },
         } as DOMMessageEvent);
 
-        return expect(promise).rejects.toEqual({ errorCode: ErrorCode.SIZE_EXCEEDED });
+        await expect(promise).rejects.toMatchObject({ errorCode: ErrorCode.SIZE_EXCEEDED });
+      });
+
+      it('videoController notifyEventToHost should fail in default version of platform', async () => {
+        expect.assertions(4); // initializeWithContext has 3 assertions + 1 local assertion
+        await mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android);
+        mobilePlatformMock.setClientSupportedSDKVersion(originalDefaultPlatformVersion);
+
+        return new media.VideoController().stop().catch(err => {
+          expect(err).toMatchObject({ errorCode: ErrorCode.OLD_PLATFORM });
+        });
+      });
+
+      it('videoController notifyEventToHost is handled successfully', async () => {
+        await mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android);
+        mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+
+        const stopPromise = new media.VideoController().stop();
+        const message = mobilePlatformMock.findMessageByFunc('media.controller');
+        expect(message).not.toBeNull();
+        expect(message.args.length).toBe(1);
+        const callbackId = message.id;
+        mobilePlatformMock.respondToMessage({
+          data: {
+            id: callbackId,
+            args: [undefined],
+          },
+        } as DOMMessageEvent);
+        expect(stopPromise).resolves;
+      });
+
+      it('videoController notifyEventToHost is not handled successfully but stop resolves', async () => {
```

Comment:
I think this test should be making sure the right exception comes out of stop when it resolves